### DISCUSSION TITLE
How to Preview OG Image Locally

### DISCUSSION THREAD

**Discussion by ephraimduncan** [vercel/next.js#54807] [Help]
### Summary

In Next.js 13, Vercel introduced a new approach to opengraph-image in the App router. More [here](opengraph-image and twitter-image). 

How do I preview the opengraph image when I'm developing locally?

Currently, I need to inspect element on the page to find the opengraph image url

**Comment by silas-v1** [comment]
type in [url]/opengraph-image

so for example http://localhost:4000/about/opengraph-image

**Comment by wmonecke** [comment]
https://chromewebstore.google.com/detail/ograph-previewer/ggcfeakcnodgcmmllfdbmngekljbhiim

use this extension. visit the url and you will be able to see the preview.

**Comment by Kedar200** [comment]
i faced the same issue actually, so i made a simple solution to this https://getlinkpeek.com/.
